content/blog/2022/12/kubewarden-2022-wrapped.md

```diff
@@ -0,0 +1,119 @@
+was during this year that many projects and organizations tackled it.
+
+The [sigstore](https://sigstore.dev) project proved to be one of the pillars of
+these initiatives.
+Kubewarden invested early in the Sigstore project, with some of our maintainers being
+active also within the Sigstore community.
+
+During the last year Kubewarden embraced Sigstore to
+[securely distribute its own assets](/blog/2022/04/securing-kubewarden-policies/),
+like container images, policies, helm charts and cli utilities.
+Moreover, we exposed Sigstore capabilities to our policy authors and provided
+a [ready to use policy](https://artifacthub.io/packages/kubewarden/verify-image-signatures/verify-image-signatures)
+that can be used to vet the container images used by Kubernetes workloads.
+
+Sigstore provides different ways to sign artifacts: with
+traditional keys,
+[keyless mode](/blog/2022/05/verifying-image-signatures/),
+[keyless mode from a GitHub Action workflow](/blog/2022/07/verify-signatures-with-gha-and-prefix/) and
+[self-signed certificates](/blog/2022/12/sigstore-certificate-verification/)
+stored on a physical token or provided by a KMS.
+During the last year we kept extending Kubewarden to ensure all these scenarios would be
+available also to our users.
+
+## Other notable features
+
+A stream of improvements have been made to the Kubewarden project during the last
+year.
+
+The Kubernetes integration was further expanded by introducing the new
+[`AdmissionPolicy`](/blog/2022/03/admission-policy/) Custom Resource. This allows
+regular Kubernetes users to define policies that are scoped to a specific
+Namespace.
+
+Next, we introduced a new operating mode for all our policies: the
+["monitor"](/blog/2022/05/monitor-mode/) mode. This allows Kubernetes administrators
+to better understand how a policy would impact their existing clusters once being
+enforced.
+
+More recently, the Kubewarden developers simplified the process of deploying
+Kubewarden inside of [air gapped](/blog/2022/11/airgap/) environments.
+
+## New policies
+
+During the last months we created many ready to use policies.
+My favorite ones are:
+
+* [Verify image signatures](https://artifacthub.io/packages/kubewarden/verify-image-signatures/verify-image-signatures): this
+  policy leverages Sigstore capabilities to ensure only trusted workloads are
+  admitted inside of your clusters
+* [Environment variables secret scanner](/blog/2022/10/env-var-secrets/): this policy finds
+  secrets, like private keys and API tokens, that are being leaked via environment
+  variables
+* [Deprecated resources](/blog/2022/11/deprecation-policy/): this policy can be
+  used to prevent the creation of resources that are deprecated by Kubernetes
+
+## See you in 2023
+
+I think one of the major changes of 2023 will be the introduction of
+[community meetings](/blog/2022/12/community-meeting/).
+The first meeting is going to take place on January 12th, 2023 at 4 PM UTC.
```

Comment:
This is maybe an overkill and you can choose to ignore this suggestion. But we should try being as timezone-agnostic as possible and include a hyperlink to a world clock that converts the time into different timezones. An example would be: https://www.worldtimebuddy.com/.
```suggestion
The first meeting will be held on January 12th, 2023 at 4 PM UTC.
```